toffy/tiling_utils_test.py

```diff
@@ -421,16 +422,16 @@ def test_generate_tiled_region_fov_list(moly_path, moly_roi_setting,
 
         # define the center points sorted
         actual_center_points_sorted = [
-            (x, y) for x in np.arange(0, 10, 5) for y in list(reversed(np.arange(85, 105, 5)))
+            (x, y) for y in list(reversed(np.arange(95, 105, 5))) for x in np.arange(0, 20, 5)
         ] + [
-            (x, y) for x in np.arange(50, 90, 10) for y in list(reversed(np.arange(140, 160, 10)))
+            (x, y) for y in list(reversed(np.arange(120, 160, 10))) for x in np.arange(50, 70, 10)
         ]
 
         # define the corresponding FOV names
         actual_fov_names = [
-            'TheFirstROI_R%dC%d' % (x, y) for y in np.arange(1, 3) for x in np.arange(1, 5)
+            'TheFirstROI_R%dC%d' % (x, y) for x in np.arange(1, 3) for y in np.arange(1, 5)
```

Comment:
Does the logic for identifying region FOV names work if there are additional underscores in the user-defined name?

Reply:
Yup, I just added that test case to `test_generate_tiled_region_fov_list`.